### ISSUE TITLE
Run vamp-test with 0.7.10 release candidates components

### ISSUE THREAD

**Issue by plamola** [magneticio/vamp#367]
As part of the release process, ensure the components for the next release pass inspection by vamp-test

- [x] create release branches for the component
- [x] have Travis-CI publish the RC artifacts
- [x] Create new local docker images using vamp-dist
- [x] Test the docker images with vamp-test

**Comment by plamola** [comment]
Output of test:
```log
[info] PulseGenericTestsSuite:
[info] Feature: We can verify some facts about the environment
[info]   Scenario: Checking if component Pulse API is of version [0.7.9.8]
[info] SavaTutorialCliSuite:
[info] Feature: The user can run through the example on the vamp.io website
[info]   I will be using the CLI 
[info]   Scenario: 1. Deploying your first blueprint
[info]     Deploying [Sava 1.0] 
[info]     Confirm [Sava 1.0] has 1 running service(s) 
[info]     Confirm [Sava 1.0] responds correctly (Routing to app works) 
[info]     Confirm [Sava 1.0] can be undeployed 
[info]     Confirm [Sava 1.0] has been removed from vamp 
[info]     Confirm [Sava 1.0] is no longer active on http://http://192.168.59.103:9050/public/ 
[info]   Scenario: 2. Doing a canary release
[info]     Deploying [Sava 1.0] 
[info]     Confirm [Sava 1.0] has 1 running service(s) 
[info]     Confirm [Sava 1.0] responds correctly (Routing to app works) 
[info]     Updating deployment to [Sava canary] 
[info]     Confirm [Sava canary] has 2 running service(s) 
[info]     Confirm [Sava canary] responds correctly (Routing to app works) 
[info]     Testing weight distribution of [Sava canary], doing a 100 requests, accepting a deviation of 10% 
[info]     - Sava Frontend 1.0 got 50 requests [50%] 
[info]     - Sava Frontend 1.1 got 50 requests [50%] 
[info]     Updating deployment to [Sava canary with filter] 
[info]     Confirm [Sava canary with filter] has 2 running service(s) 
[info]     Confirm [Sava canary with filter] responds correctly (Routing to app works) 
[info]     Confirm [Sava canary with filter] responds correctly on http://192.168.59.103:9050/public/ to Chrome 
[info]     Confirm [Sava canary with filter] responds correctly on http://192.168.59.103:9050/public/ to Firefox 
[info]     Confirm [Sava canary with filter] responds correctly on http://192.168.59.103:9050/public/ to Safari 
[info]     Confirm [Sava canary with filter] responds correctly on http://192.168.59.103:9050/public/ to IE 
[info]     Confirm [Sava canary with filter] can be undeployed 
[info]     Confirm [Sava canary with filter] has been removed from vamp 
[info]     Confirm [Sava canary with filter] is no longer active on http://http://192.168.59.103:9050/public/ 
[info]   Scenario: 3. Splitting the monolith into services
[info]     Deploying [Sava 1.2] 
[info]     Confirm [Sava 1.2] has 3 running service(s) 
[info]     Confirm [Sava 1.2] responds correctly (Routing to app works) 
[info]     Confirm [Sava 1.2] can be undeployed 
[info]     Confirm [Sava 1.2] has been removed from vamp 
[info]     Confirm [Sava 1.2] is no longer active on http://http://192.168.59.103:9060/public/ 
[info]   Scenario: 4. Merging a changed topology *** FAILED ***
[info]   Could not detect services: Futures timed out after [15 seconds] (DeploymentTest.scala:72)
[info]     Deploying [Sava 1.2] 
[info]     Confirm [Sava 1.2] has 3 running service(s) 
[info]     Confirm [Sava 1.2] responds correctly (Routing to app works) 
[info]     Updating deployment to [Sava 1.2 merged with 1.3] 
[info]     Confirm [Sava 1.2 merged with 1.3] has 5 running service(s) 
[info]     Confirm [Sava 1.2 merged with 1.3] responds correctly (Routing to app works) 
[info]     Updating deployment to [Sava 1.3 - start ab-test] 
[info]     Confirm [Sava 1.3 - start ab-test] has 5 running service(s) 
[info]     Confirm [Sava 1.3 - start ab-test] responds correctly (Routing to app works) 
[info]     Testing weight distribution of [Sava 1.3 - start ab-test], doing a 100 requests, accepting a deviation of 10% 
[info]     - Sava Frontend 1.2 got 50 requests [50%] 
[info]     - Sava Frontend 1.3 got 50 requests [50%] 
[info]     Updating deployment to [Sava 1.3 - end ab test] 
[info]     Confirm [Sava 1.3 - end ab test] has 5 running service(s) 
[info]     Confirm [Sava 1.3 - end ab test] responds correctly (Routing to app works) 
[info]     Removing part of deployment with [Sava 1.2 remove] 
[info]     Confirm [Sava 1.3] has 2 running service(s) 
[info] Pre test cleanup 
[info] - Removing 4 blueprints 
[info] - Removing 7 breeds 
[info] CliDeploymentSuite:
[info] CoreYamlGenericTestsSuite:
[info] Feature: We can verify some facts about the environment
[info]   Scenario: Checking if component Rest API [yaml] is of version [0.7.9.8]
[info] CliGenericSuite:
[info] - should return the correct version number
[info] - should be able to return the info structure
[info] - should be able to return a help message
[info] AggregationSuite:
[info] - should be able to aggregate metrics
[info] - should be able to aggregate events !!! IGNORED !!!
[info] StorageSuite:
[info] - should be able to send a metric to pulse
[info] - should be able to query recently sent metric from pulse
[info] - should be able to send a json blob to pulse
[info] - should be able to query recently sent blob from pulse !!! IGNORED !!!
[info] - should be able to send a typed event to .pulse
[info] - should be able to query recently sent event from pulse
[info] RouterGenericTestsSuite:
[info] Feature: We can verify some facts about the environment
[info]   Scenario: Checking if component Router is of version [0.7.9]
[info] CliYamlGenericTestsSuite:
[info] Feature: We can verify some facts about the environment
[info]   Scenario: Checking if component CLI is of version [0.7.9.5]
[info] SavaTutorialRestSuite:
[info] Feature: The user can run through the example on the vamp.io website
[info]   I will be using the Rest API [yaml] 
[info]   Scenario: 1. Deploying your first blueprint
[info]     Deploying [Sava 1.0] 
[info]     Confirm [Sava 1.0] has 1 running service(s) 
[info]     Confirm [Sava 1.0] responds correctly (Routing to app works) 
[info]     Confirm [Sava 1.0] can be undeployed 
[info]     Confirm [Sava 1.0] has been removed from vamp 
[info]     Confirm [Sava 1.0] is no longer active on http://http://192.168.59.103:9050/public/ 
[info]   Scenario: 2. Doing a canary release
[info]     Deploying [Sava 1.0] 
[info]     Confirm [Sava 1.0] has 1 running service(s) 
[info]     Confirm [Sava 1.0] responds correctly (Routing to app works) 
[info]     Updating deployment to [Sava canary] 
[info]     Confirm [Sava canary] has 2 running service(s) 
[info]     Confirm [Sava canary] responds correctly (Routing to app works) 
[info]     Testing weight distribution of [Sava canary], doing a 100 requests, accepting a deviation of 10% 
[info]     - Sava Frontend 1.0 got 50 requests [50%] 
[info]     - Sava Frontend 1.1 got 50 requests [50%] 
[info]     Updating deployment to [Sava canary with filter] 
[info]     Confirm [Sava canary with filter] has 2 running service(s) 
[info]     Confirm [Sava canary with filter] responds correctly (Routing to app works) 
[info]     Confirm [Sava canary with filter] responds correctly on http://192.168.59.103:9050/public/ to Chrome 
[info]     Confirm [Sava canary with filter] responds correctly on http://192.168.59.103:9050/public/ to Firefox 
[info]     Confirm [Sava canary with filter] responds correctly on http://192.168.59.103:9050/public/ to Safari 
[info]     Confirm [Sava canary with filter] responds correctly on http://192.168.59.103:9050/public/ to IE 
[info]     Confirm [Sava canary with filter] can be undeployed 
[info]     Confirm [Sava canary with filter] has been removed from vamp 
[info]     Confirm [Sava canary with filter] is no longer active on http://http://192.168.59.103:9050/public/ 
[info]   Scenario: 3. Splitting the monolith into services
[info]     Deploying [Sava 1.2] 
[info]     Confirm [Sava 1.2] has 3 running service(s) 
[info]     Confirm [Sava 1.2] responds correctly (Routing to app works) 
[info]     Confirm [Sava 1.2] can be undeployed 
[info]     Confirm [Sava 1.2] has been removed from vamp 
[info]     Confirm [Sava 1.2] is no longer active on http://http://192.168.59.103:9060/public/ 
[info]   Scenario: 4. Merging a changed topology
[info]     Deploying [Sava 1.2] 
[info]     Confirm [Sava 1.2] has 3 running service(s) 
[info]     Confirm [Sava 1.2] responds correctly (Routing to app works) 
[info]     Updating deployment to [Sava 1.2 merged with 1.3] 
[info]     Confirm [Sava 1.2 merged with 1.3] has 5 running service(s) 
[info]     Confirm [Sava 1.2 merged with 1.3] responds correctly (Routing to app works) 
[info]     Updating deployment to [Sava 1.3 - start ab-test] 
[info]     Confirm [Sava 1.3 - start ab-test] has 5 running service(s) 
[info]     Confirm [Sava 1.3 - start ab-test] responds correctly (Routing to app works) 
[info]     Testing weight distribution of [Sava 1.3 - start ab-test], doing a 100 requests, accepting a deviation of 10% 
[info]     - Sava Frontend 1.2 got 50 requests [50%] 
[info]     - Sava Frontend 1.3 got 50 requests [50%] 
[info]     Updating deployment to [Sava 1.3 - end ab test] 
[info]     Confirm [Sava 1.3 - end ab test] has 5 running service(s) 
[info]     Confirm [Sava 1.3 - end ab test] responds correctly (Routing to app works) 
[info]     Removing part of deployment with [Sava 1.2 remove] 
[info]     Confirm [Sava 1.3] has 2 running service(s) 
[info]     Confirm [Sava 1.3] responds correctly (Routing to app works) 
[info]     Confirm [Sava 1.3] can be undeployed 
[info]     Confirm [Sava 1.3] has been removed from vamp 
[info]     Confirm [Sava 1.3] is no longer active on http://http://192.168.59.103:9060/public/ 
[info] Pre test cleanup 
[info] - Removing 4 blueprints 
[info] - Removing 7 breeds 
[info] CliArtifactSuite:
[info] Run completed in 3 minutes, 59 seconds.
[info] Total number of tests run: 21
[info] Suites: completed 12, aborted 0
[info] Tests: succeeded 20, failed 1, canceled 0, ignored 2, pending 0
[info] *** 1 TEST FAILED ***
[error] Failed tests:
[error] 	io.vamp.test.scenario.sava.implementation.SavaTutorialCliSuite
[error] (test:test) sbt.TestsFailedException: Tests unsuccessful
[error] Total time: 242 s, completed Aug 26, 2015 12:29:26 PM
```

**Comment by plamola** [comment]
The one failing test is due to missing functionality in Vamp CLI; see #364
